sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/README.md

```diff
@@ -0,0 +1,79 @@
+# CoreWCF Azure Queue Storage library
+
+CoreWCF Azure Queue Storage is the service side library that will help existing WCF services to be able to use Azure Queue Storage to communicate with clients as a modern replacement to using MSMQ.
+
+## Getting started
+
+### Install the package
+
+Install the Microsoft.CoreWCF.Azure.StorageQueues library for .NET with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Microsoft.CoreWCF.Azure.StorageQueues
+```
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
+[Storage Account][storage_account_docs] to use this package.
+
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+### Authenticate the service to Azure
+
+In order to receive requests from the Azure Queue Storage service, you'll need to configure CoreWCF with the appropriate endpoint and credentials.  The [Azure Identity library][identity] makes it easy to add Azure Active Directory support for authenticating with Azure services.
```

Comment:
Azure Active Directory is no more.   These will need to reflect the "Entra ID" branding.

Reply:
This language was copied from the Azure Storage Queue readme documentation [located here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage/Azure.Storage.Queues#authenticate-the-client). I wanted to match the style for how it was referenced. Do you have either a link to usage guidelines for how to refer to Entra ID, or ideally if the Azure Storage Queues readme is updated, we can copy from there.

Reply:
@scottaddie :  You're the expert on the rebranding use.   Can you offer thoughts?

Reply:
What do you want to do about this?

Reply:
We need to update, as there's no such thing as AAD.   We should also poke the Storage team to update their docs.

//cc @scottaddie , @schaabs

Reply:
See the guidance at https://learn.microsoft.com/azure/active-directory/fundamentals/how-to-rename-azure-ad for the rebranding.